pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1048,14 +1053,28 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 	return &cfg, nil
 }
 
+// getVolumeID returns volumeID from the PV or volumename.
+func getVolumeID(pv *v1.PersistentVolume, volumeName string) string {
```

Comment:
I'd prefer to return `(string, error)`.

And the error could be `"failed to get volume ID"`.